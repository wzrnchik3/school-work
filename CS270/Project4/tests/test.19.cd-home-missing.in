setenv HOME 
cd /usr/bin
pwd
cd
cd ..
echo should be in /usr still
pwd
