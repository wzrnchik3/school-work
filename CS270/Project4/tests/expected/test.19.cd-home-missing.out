/usr/bin
should be in /usr still
/usr
